src/hedera/hederaInfo.js

```diff
@@ -0,0 +1,57 @@
+/* global */
+// @flow
+
+import { HbarUnit } from '@hashgraph/sdk'
+import { type EdgeCurrencyInfo } from 'edge-core-js/types'
+
+// import { imageServerUrl } from '../common/utils'
+import { type HederaSettings } from './hederaTypes.js'
+
+export const useTestnet = true
+
+const otherSettings: HederaSettings = {
+  creatorApiServers: useTestnet
+    ? ['http://10.0.2.2:8080'] // ? ['https://creator.testnet.myhbarwallet.com']
```

Comment:
Will fix, had to make changes to these services during development.

Reply:
On this note, I didn't see much of a convention for switching e.g. the Ethereum plugin to use testnets, besides a comment on the `chainId` parameter here: https://github.com/EdgeApp/edge-currency-accountbased/blob/a7e3b9f4162b42e33cb01d355b932174b9c154f3/src/ethereum/ethEngine.js#L532

Would you prefer if I remove the ternary and leave it on the mainnet values for merge?

Reply:
If you look at the recent architecture of the Ethereum plugin, it can export multiple currency types with the only difference being the API servers and chain ID. IT would show up as a different wallet type that a user can create, but this is one way to expose testnet.

Reply:
That would be pretty interesting, although on testnet we still have the issue of account activation. We could support imports only for testnet wallets (since [the Hedera portal](https://portal.hedera.com/) automatically creates testnet accounts for registered users and tops them up daily with HBAR).